ietf-microwave-radio-link.yang

```diff
@@ -493,19 +594,27 @@ module ietf-microwave-radio-link {
              Adaptive coding/modulation shall not go
              above this value.";
         }
-        leaf reference-modulation {
-          type identityref {
-            base mw-types:coding-modulation;
-          }
-          description
-            "the reference-modulation is the coding modulation to be used in the reference mode, as defined in ETSI EN 302 217-1 V3.2.2 (2020-02). 
-			The reference mode identifies the operative mode which characteristics (i.e. system capacity, spectral efficiency class over a given channel separation) 
-			are used (i.e. declared in the licensing process) in the link per link coordination analysis.
-			Its value has to be between the 'selected-min-acm' and the 'selected-max-acm' attribute's values.
-			";
-        }
```

Comment:
Why this attribute has been removed in this PR (which is intended to add only header compression attributes)?

I am wondering whether some of the previous changes to the main branch got lost